src/common/LocationNotice.tsx

```diff
@@ -15,11 +15,11 @@ export const LocationNotice: FunctionComponent<LocationNoticeProps> = ({
       {instructions.includes(Instruction.walkIn) && (
         <div>
           <br />
-          <p>{t("walkins.noticeList.text")}</p>
+          <p>{t("walkins.healthpointModal.text")}</p>
```

Comment:
@gpoliko does this mean we will need to change keys for every translation json? 

I think even if the JSON structure is not tidy anymore we should keep the same keys as long as they are the same text.

Reply:
@gpoliko - typically, even if the text is the same, they are still two different sections and we should have two separate keys for them. Otherwise we risk changing the text for one (possibly unrelated) component and inadvertently update it somewhere else too.

Reply:
@noway @olafwrieden points taken, and makes complete sense. I'll revert this change 👍🏾